lua/entities/gmod_wire_rt_screen.lua

```diff
@@ -116,7 +116,10 @@ if SERVER then
 
         local Inputs = table.Copy(InputsTable)
 
-        for _, tbl in pairs(GetMaterialParameters(self:GetScreenMaterial())) do
+		local matParams = GetMaterialParameters(self:GetScreenMaterial())
```

Comment:
This code will leave screen with bad material partially-initialized (no inputs will be added).
Maybe it is better to remove broken screens completely?